src/main/java/com/neokortex/task/Task.java

```diff
@@ -0,0 +1,80 @@
+package com.elsria.task;
+
+import com.elsria.exceptions.InvalidTaskSerializationException;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+public abstract class Task {
+    private static final char markedSymbol = 'X';
+    private static final char unmarkedSymbol = ' ';
+    protected String description;
+    protected boolean isMarked;
```

Comment:
Fields/methods should default to private unless wider visibility is necessary. This tightens encapsulation and prevents accidental misuse.
```suggestion
    private String description;
    private boolean isMarked;
```